lib/Doctrine/Common/Cache/RedisCache.php

```diff
@@ -15,18 +15,24 @@
  */
 class RedisCache extends CacheProvider
 {
+    public const SERIALIZER_PHP = 1;
+    public const SERIALIZER_IGBINARY = 2;
+
     /** @var Redis|null */
     private $redis;
 
+    /** @var int */
```

Comment:
This should be documented as nullable since the value is only configured when `setRedis()` gets called.

Reply:
I'll fix this.